### ISSUE TITLE
Using Input component with https://react-hook-form.com/

### ISSUE THREAD

**Issue by yalamber** [Telegram-Mini-Apps/TelegramUI#56]
When using Input component with react-hook-form we are receiving below error. 
Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

**Comment by yalamber** [comment]
What's the difference between FormInput and Input, It seems like FormInput is not exported.

**Comment by batFormat** [comment]
You can try it like this

```
 <Controller
      name="answer"
      control={control}
      render={({ field: { value, onChange } }) => (
        <Input value={value ?? ''} onChange={onChange} />
      )}
    />
```

**Comment by mainsmirnov** [comment]
Thank you, issue was fixed in `2.1.8` version
